src/components/Panels/PanelHeader.vue

```diff
@@ -0,0 +1,13 @@
+<template>
```

Comment:
if this is a sharable component between the Panels, let's create a Panels/shared & move it to there, same flow for any shareable file.

Reply:
moved PanelHeader into a folder named shared.

Will also have to make a separate sharable component for `<v-expansion-panel-text>` 
Since it's repeated twice in `ElementsPanel.vue`
And maybe needed once again in `LayoutPanel.vue`